lib/mix_gleam/config.ex

```diff
@@ -50,7 +50,7 @@ defmodule MixGleam.Config do
   # Run "mix help compile.app" to learn about applications.
   def application do
     [
-      extra_applications: [:logger]
+      extra_applications: [:logger, :eunit]
```

Comment:
That is true. It occurred to me after raising the PR.  This might be needed. I wanted to write the tests in gleam for the gleam code. I can try the alternatives.

Reply:
If you write tests in Gleam and run eunit using the Mix eunit it will still run them 👍

Reply:
Yeah I will do that. Thanks.